debian/jellyfin.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description = Jellyfin Media Server
-After = network.target
+After = network-online.target
```

Comment:
For maximum compatibility shouldn't it be `After = network.target network-online.target`?

Reply:
I think that makes sense, I'm not sure offhand what the interdependency of those two is.

Reply:
https://www.freedesktop.org/software/systemd/man/systemd.special.html#network-online.target

Not sure what the implications of using `network-online,target` is on offline systems with only localhost.